service/src/test/resources/data/published_requests/outgoing_mocked_request2b_all_pairs.json

```diff
@@ -267,7 +267,7 @@
             "cmoSampleClass": "Normal", 
             "cmoSampleName": "C-PPPXX2-N001-d", 
             "collectionYear": "", 
-            "igoId": "22022_BZ_4", 
```

Comment:
Shouldn't some of these remain as igoId to test if the json is being processed correctly and successfully?